src/ontology/cl-edit.owl

```diff
@@ -32517,6 +32518,19 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
 SubClassOf(obo:CL_4042027 obo:CL_0002614)
 SubClassOf(obo:CL_4042027 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001903))
 
+# Class: obo:CL_4042031 (immune oligodendroglia)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30747918") Annotation(oboInOwl:hasDbXref "PMID:35301426") Annotation(oboInOwl:hasDbXref "PMID:35615276") Annotation(oboInOwl:hasDbXref "PMID:35662200") obo:IAO_0000115 obo:CL_4042031 "An oligodendrocyte of the central nervous system that exhibits immune properties such as non-self antigen presentation to T cells, phagocytosis of debris, and cytokine and chemokine production. This cell type is immunoreactive during inflammation, neurodegenerative disorders, chronic stress and major depressive disorder.")
+AnnotationAssertion(terms:contributor obo:CL_4042031 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042031 "2024-10-11T12:51:17Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35615276") oboInOwl:hasExactSynonym obo:CL_4042031 "immune oligodendrocyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35301426") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042031 "Im-OL")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35615276") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042031 "ImOL")
```

Comment:
This article does a very bad job following consistency on this abbreviation, so it also have the non-capitalized version. Consider including it as well

Reply:
I do not find the non capitalised version, could you show me?

Reply:
I think that as you pushed some commits afterwards, the comment went to a different line. The comment was for imOLG/ImOLG that is found in [PMID:30747918](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6544546/). If you want to address comments in different commits, it is better that you don't push the branch until all comments are addressed, otherwise they don't match with the appropriate line.

Reply:
just addressed the comment you will find it in the next commit